modules/api/src/main/java/io/fluo/api/client/FluoAdmin.java

```diff
@@ -24,7 +24,7 @@
   /**
    * Specifies Fluo initialization options such as clearing Zookeeper or existing Accumulo table.
    */
-  public static class InitOpts {
+  class InitOpts {
```

Comment:
yes it has to be static so its redundant:  http://stackoverflow.com/questions/71625/why-would-a-static-inner-interface-be-used-in-java